assets/model_monitoring/components/model_performance/model_performance_compute_metrics/spec.yaml

```diff
@@ -0,0 +1,78 @@
+$schema: http://azureml/sdk-2-0/SparkComponent.json
+type: spark
+
+name: model_performance_compute_metrics
+display_name:  Model Performance - Compute Metrics
+description: Compute model performance metrics leveraged by the model performance monitor.
+version: 0.0.2
+is_deterministic: true
+
+code: ../../src
+entry:
+  file: ./model_performance_metrics/compute_metrics.py
```

Comment:
We usually make run.py as the entrypoint script, which I believe is what you are doing. Then here should be run.py.